### ISSUE TITLE
'id' property missing when making changes to company users or company settings

### ISSUE THREAD

**Issue by PatrickA-RiseVision** [Rise-Vision/rise-vision-apps#2838]
### What is the expected behavior?
Users with the administrator permission can add/remove users, and modify company settings.

### What is the actual behavior?
Under some circumstances they get this error:

'id' property missing

### What steps can be taken to replicate the problem?
Not sure, I can't replicate.  We've had 2 users bring it up.  I can share usernames privately.

**Comment by ezequielc** [comment]
@PatrickA-RiseVision @alex-deaconu I can't replicate this both with a Rise Admin or a regular user account.
But from one of the tickets screenshot, I did notice that the company was not loaded (i.e. company data request failed), as the company name was blank below their username. I tried to block such requests and I manage to get the same outcome: company settings was blank and I got the same error as reported when trying to save the company.

In summary, I think this is due to a network failure (we noticed instability in our infrastructure today) but we definitely need to handle this situation better, as other areas of the app could also be affected. Maybe by showing an error message and/or forcing a page reload. Thanks

**Comment by alex-deaconu** [comment]
@PatrickA-RiseVision @ezequielc we've deployed a correction for this issue. When the User's Company fails to load, we will perform the same action as when the User's Info fails to load. That is, we will log the user out and redirect them to the login page. An error will be shown and the inconvenience is they will have to log in again. Hopefully this does not happen too often, but we have added some event tracking to see if it does. Closing for now. Thanks!
